src/cubic/cubic_roots.h

```diff
@@ -0,0 +1,48 @@
+/*
+  General root calculator for cubic equations of state.
+
+  Author: John Eslick
+  Date: November 22, 2016
+
+  Notes:
+*/
+#include <time.h>
+#include <math.h>
+#include "funcadd.h"
+#undef fprintf
```

Comment:
The ASL stuff is weird (in my opinion).  It defines an fprintf macro.  I think I had to do the undef to keep it from messing up other things.  Since I get the question from every one that actually looks at the code, I probably should add a comment.

Reply:
Found a source file, I wasn't using anymore, and it turns out I don't need this anymore.  So is fixed.